### PR TITLE
Gives tooltip when using the nuget

### PR DESCRIPTION
**Pull request by jphellemons**
My first PR to add a tooltip when you are working with the 2.x beta Nuget. Makes it a bit more clear not to use paypalenvironment directly. Because there are no docs yet.
4w6a7sg

### CODE REVIEW

**Comment by jphellemons** [comment]
Should I work in the 2.0-beta branch or the develop branch?

File: Source/v1/Payments/PaymentCreateRequest.cs [hunk 1]
```diff
@@ -15,7 +15,11 @@
 namespace PayPal.v1.Payments
 {
     /// <summary>
-    /// Creates a sale, an authorized payment to be captured later, or an order. To create a sale, authorization, or order, include the payment details in the JSON request body. Set the `intent` to `sale`, `authorize`, or `order`. Include payer, transaction details, and, for PayPal payments only, redirect URLs. The combination of the `payment_method` and `funding_instrument` determines the type of payment that is created. For more information, see [Payments REST API](/docs/integration/direct/payments/).<blockquote><strong>Note:</strong> Authorizations are guaranteed for up to three days, though you can attempt to capture an authorization for up to 29 days. After the three-day honor period authorization expires, you can [reauthorize](#authorization_reauthorize) the payment.</blockquote>
+    /// Creates a Sale, an authorized Payment to be captured later, or an Order. To create a Sale, Authorization, or Order, include the payment details in the JSON request body.
```
Comment (jaypatel512) [inline Source/v1/Payments/PaymentCreateRequest.cs]: Hey @jphellemons ! These files are autogenerated using our SDK. Any changes done here will be overwritten when the generator is ran again.

**Comment by jaypatel512** [comment]
I have cherry-picked your [first commit](https://github.com/paypal/PayPal-NET-SDK/commit/7c660ea69e89e4c2badf16c1281245c1281e7bb2), to make the tooltip change. Only the files inside `Core` directory are hand written, and can be changed. Anything inside `v1` directory will be automatically re-generated. I am going to close this PR.

**Comment by jphellemons** [comment]
Thank you for your explanation! I was worried that I might be using in the wrong branch? '2.0-beta' or 'develop' branch.

**Comment by jaypatel512** [comment]
'2.0-beta' is the correct branch for 2.0-beta development. The `develop` branch is our current stable 1.x SDK

**Pull request closed without merging**